aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -166,6 +166,35 @@ func resourceAwsApiGatewayRestApiUpdateOperations(d *schema.ResourceData) []*api
 		})
 	}
 
+	if d.HasChange("binary_media_types") {
+		o, n := d.GetChange("binary_media_types")
+		prefix := "binaryMediaTypes"
+
+		old := o.([]interface{})
+		new := n.([]interface{})
+
+		// Remove every binary media types. Simpler to remove and add new ones,
+		// since there are no replacings.
+		for _, v := range old {
+			m := v.(string)
+			operations = append(operations, &apigateway.PatchOperation{
+				Op:   aws.String("remove"),
+				Path: aws.String(fmt.Sprintf("/%s/%s", prefix, strings.Replace(m, "/", "~1", -1))),
```

Comment:
Do you think it's worth decoupling the escaping logic into a separate function, e.g. to `structures.go`? I did something very similar in the K8S provider:
https://github.com/terraform-providers/terraform-provider-kubernetes/commit/c2d73ca48914b01936b87e1ce6d37d777719089e